### DISCUSSION TITLE
Add a page to see a list of notes added for a specific habit

### DISCUSSION THREAD

**Discussion by shiv19** [iSoron/uhabits#1896] [Feature Requests]
Currently when I add a note to a habit it adds a dot on that day, and you can tap the check mark to view the note.
Or open the habit details page, tap edit on the calendar, and tap on a day with a dot to see that note.

I propose that we add another page in the habit detail page which lists all the notes added for that habit in a chronological order. And perhaps let you search through the notes.

**Comment by herrbrixter** [comment]
Duplicate of #1601
